katacomb/katacomb/mock_dataset.py

```diff
@@ -456,19 +454,19 @@ def _generate_ref_ant_compound_scans():
 
         # Labels seem to only involve tracks, separate them out
         label_scans = [tup for tup in ref_ant_compound_scans if tup[1] == 'track']
-        events, values, _ = zip(*label_scans)
+        events, values, _ = list(zip(*label_scans))
```

Comment:
I suspect the `list` might not be needed, because the multiple variables on the left hand side will make it iterate over the generator. But please double-check. Also applies a few times below.

Reply:
Yes- it works. I think I just missed these ones when I was going through and removing all the `list()` calls before.